packages/twenty-front/src/modules/workflow/components/RightDrawerWorkflowEditStepContent.tsx

```diff
@@ -33,7 +36,9 @@ const getStepDefinitionOrThrow = ({
   }
 
   if (!isDefined(currentVersion.steps)) {
-    throw new Error('Expected to find an array of steps');
+    throw new Error(
+      'Expected to find an array of steps while trying to locate a specific step',
```

Comment:
I'm not sure to get this error :p
Maybe: Could not load workflow steps?

Reply:
Changed for "Expected the steps to be defined while trying to locate a specific step".

Reply:
I think this should be more user understandable, it's an open topic that @ijreilly already brought

Maybe: "Malformed workflow version : missing steps"

Reply:
I'm just wondering: are steps optional? why do we actually need to check their existence from a typing perspective?

Reply:
Steps are `null` by default, which is why I wrote this assertion. However, we might want to set this field's default value to an empty array.

Reply:
Changed the error message to "Malformed workflow version: missing steps information; be sure to create at least one step before trying to edit one"